### ISSUE TITLE
[Feature Request] Add per-package pacman argument

### ISSUE THREAD

**Issue by CupricReki** [Lukas-Heiligenbrunner/AURCache#65]
I would imagine this working by adding an 'Edit' action where you can add common arguments by check boxes, and a text box for adding custom arguments. Would also help solve #64 as you could specify a pgp key. 

A few useful pacman argument:

- --skippgpcheck (when pgp keys don't exist)
- --noconfirm (for package conflicts)

**Comment by Lukas-Heiligenbrunner** [comment]
Hey, thats a good idea. 
--noconfirm is already added to every build. See the current command here: https://github.com/Lukas-Heiligenbrunner/AURCache/blob/master/backend/src/builder/build.rs#L255

**Comment by CupricReki** [comment]
Hmm. I still get this error when trying to build `obs-studio-tytan652`
```
:: There are 2 providers available for jack:
:: Repository extra:
    1) jack2  2) pipewire-jack  
Enter a number (default=1): 

:: There are 3 providers available for opengl-driver:
:: Repository extra:
    1) mesa  2) mesa-amber  3) nvidia-utils  
Enter a number (default=1): 
:: Calculating conflicts...
:: Calculating inner conflicts...

:: Inner conflicts found:
    ffmpeg: ffmpeg-obs (ffmpeg)  

:: Conflicting packages will have to be confirmed manually
error: can not install conflicting packages with --noconfirm
No files found in build directory
```
Log attached
[22d1e9294cfd91d71493e992e2d7903460955986b97f2af3fda4fde1694a7d84-json.log](https://github.com/user-attachments/files/16838882/22d1e9294cfd91d71493e992e2d7903460955986b97f2af3fda4fde1694a7d84-json.log)

**Comment by Lukas-Heiligenbrunner** [comment]
fixed in #74
